src/messages.cpp

```diff
@@ -90,93 +90,96 @@ struct game_message : public JsonDeserializer, public JsonSerializer {
     }
 };
 
-bool message_exceeds_ttl(const game_message &message) {
-    return message_ttl > 0 && message.timestamp_in_user_actions + message_ttl <= g->get_user_action_counter();
+bool message_exceeds_ttl( const game_message &message )
+{
+    return message_ttl > 0 &&
+           message.timestamp_in_user_actions + message_ttl <= g->get_user_action_counter();
 }
 
 } //namespace
 
-class Messages::impl_t {
-public:
-    std::deque<game_message> messages;   // Messages to be printed
-    int                      curmes = 0; // The last-seen message.
-
-    bool has_undisplayed_messages() const {
-        return !messages.empty() && messages.back().turn() > curmes;
-    }
-
-    game_message const& history(int const i) const {
-        return messages[messages.size() - i - 1];
-    }
+class Messages::impl_t
+{
+    public:
+        std::deque<game_message> messages;   // Messages to be printed
+        int                      curmes = 0; // The last-seen message.
```

Comment:
```
23:23 < FRS> Toothspit: huh. That giant space, y'know- that might be astyle trying to align lvalue names
23:23 < Toothspit> FRS: Ohhhh u rite
23:23 < Toothspit> Still looks goofy =/
23:24 < FRS> Rather, though I figure this is what's considered 'correct' if I were to follow CODE_STYLE.md word-for-word.
23:25 < Toothspit> Right, I saw it was like that before anyway
23:25 < FRS> Shall I drop in these few lines of IRC and conclude "This is correct according to astyle, a coredev can override if needed"?
23:25 < Toothspit> Sure mang
```
"case closed" I figure(?) Unless people have substantial objections, I figure this is correct.